### PR TITLE
Added timestamp prefix to all lines in output from pytest process

### PR DESCRIPTION
**Pull request by tomaszklak**
### Problem
Not every output line produced by pytest (or its child processes) has a timestamp prefix.

### Solution
This solution prepends a timestamp to all lines produced by pytest (or its child processes). To make sure that we add a timestamp right after the line was produced both output buffering and input buffering has to be disabled (by setting `PYTHONUNBUFFERED` to `1` and `bufsize=0` respectively). Without this a 10-15s delay is observed.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Comment by matislovas** [comment]
There's a bunch of `datetime.now()` in `print` in `telio.py`. Shouldn't we cleanup those as well ..?

**Comment by matislovas** [comment]
+1

File: nat-lab/run_local.py [hunk 1]
```diff
@@ -5,21 +5,55 @@
...
 from typing import List, Optional, Dict, Any
 
 PROJECT_ROOT = os.path.normpath(os.path.dirname(os.path.realpath(__file__)) + "/../..")
 
 TEST_TIMEOUT = 120
 
 
+def add_timestamps_to_output_lines(input_stream, output_stream):
+    for line in input_stream:
+        line = line.decode("ascii").rstrip()
```
Comment (packgron) [inline nat-lab/run_local.py:19]: Are you sure we will not have utf8 bytes here?

Comment (tomaszklak) [reply nat-lab/run_local.py:19]: To be honest I have no idea, but utf8 is indeed safer. I will change this to utf8.

**Review by packgron — APPROVED** [review]
+1

**Pull request merged**